Source/Extensions/AutolayoutManager.swift

```diff
@@ -0,0 +1,130 @@
+import UIKit
+
+// MARK: - BottomContainer autolayout
+
+extension BottomContainerView {
+
+  func setupConstraints() {
+
+    for attribute: NSLayoutAttribute in [.CenterX, .CenterY] {
```

Comment:
Of course, I understand, why all the hassle with not adding dependencies, but after trying Masonry/Cartography default  AutoLayout syntax looks really weird.

Nice idea to keep it away from logic-related code.